### PR TITLE
[AIRFLOW-6770] Run particular test using breeze CLI bug fix

### PR DESCRIPTION
**Pull request by jithin-sukumar**
### Description
Running a particular test from breeze CLI runs all the tests within _tests/_ directory instead of running only the tests specified by **test-target** flag

**Before:**

```
⇒  ./breeze --test-target tests/test_core.py
.
.
.
Starting the tests with those pytest arguments: tests/

======================================================================================== test session starts ========================================================================================
platform linux -- Python 3.6.10, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow, inifile: pytest.ini
plugins: requests-mock-1.7.0, cov-2.8.1, flaky-3.6.1, instafail-0.4.1.post0, celery-4.4.0
collected 5055 items / 1 skipped / 5054 selected
```

**After:**

```
⇒  ./breeze --test-target tests/test_core.py
.
.
.
Starting the tests with those pytest arguments: tests/test_core.py

======================================================================================== test session starts ========================================================================================
platform linux -- Python 3.6.10, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow, inifile: pytest.ini
plugins: requests-mock-1.7.0, cov-2.8.1, flaky-3.6.1, instafail-0.4.1.post0, celery-4.4.0
collected 24 items 
```
---
Issue link: [AIRFLOW-6770](https://issues.apache.org/jira/browse/AIRFLOW-6770)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ jobs:
         ENABLED_INTEGRATIONS="cassandra kerberos mongo openldap rabbitmq redis"
         RUN_INTEGRATION_TESTS=all
       stage: test
-      script: ./scripts/ci/ci_run_airflow_testing.sh tests/tests_core.py
+      script: ./scripts/ci/ci_run_airflow_testing.sh tests/test_core.py
```
Comment (potiuk) [inline .travis.yml]:
```suggestion
```
That's an interesting one. Actually it's a mistake here. We should not constraint ourselves in this test only to test_core. It is likely a remnant from some testing done when I was implementing it. Good that you noticed it.

File: scripts/ci/in_container/entrypoint_ci.sh [hunk 2]
```diff
@@ -211,6 +211,10 @@ fi
 
 TEST_DIR="tests/"
```
Comment (potiuk) [inline scripts/ci/in_container/entrypoint_ci.sh]:
```suggestion
TESTS_TO_RUN="tests/"
```

Comment (potiuk) [reply scripts/ci/in_container/entrypoint_ci.sh]: I think this will be better name :)

File: scripts/ci/in_container/entrypoint_ci.sh [hunk 3]
```diff
@@ -211,6 +211,10 @@ fi
...
 
+if [[ ${#@} -gt 0 && -n "$1" ]]; then
+    TEST_DIR="$1"
+fi
+
 if [[ -n ${RUNTIME} ]]; then
     CI_ARGS+=("--runtime" "${RUNTIME}" "-rpfExX")
     TEST_DIR="tests/runtime"
```
Comment (potiuk) [inline scripts/ci/in_container/entrypoint_ci.sh]:
```suggestion
    TESTS_TO_RUN="tests/runtime"
```

Comment (potiuk) [reply scripts/ci/in_container/entrypoint_ci.sh]: And also we should change TEST_DIR below to TESTS_TO_RUN :)

**Review by potiuk — CHANGES_REQUESTED** [review]
Good  catch @jithin97 ! I looked at it and made a few suggestions :)

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Thanks @jithin97 !

File: scripts/ci/in_container/entrypoint_ci.sh (hunk 3 above)
Comment (jithin-sukumar) [reply scripts/ci/in_container/entrypoint_ci.sh]: fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0

File: scripts/ci/in_container/entrypoint_ci.sh (hunk 2 above)
Comment (jithin-sukumar) [reply scripts/ci/in_container/entrypoint_ci.sh]: fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0

File: .travis.yml (hunk 1 above)
Comment (jithin-sukumar) [reply .travis.yml]: fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0

**Pull request merged**